components/core/src/clp_s/TimestampDictionaryWriter.hpp

```diff
@@ -47,9 +49,9 @@ class TimestampDictionaryWriter {
      * @return the epoch time corresponding to the string timestamp
      */
     epochtime_t ingest_entry(
```

Comment:
Not mandatory, but since it seems like a small change, do you want to change the callers in ArchiveWriter to also use string_views?